.env.example

```diff
@@ -1 +1 @@
-API_URL=
```

Comment:
why it was renamed?

Reply:
Vite needs VITE prefix to env variables to download them automatically with no need to add some additional configs, and I decided to use this feature of Vite

Reply:
got you